packages/homepage/content/articles/building-accessible-apps-with-speed-using-chakra-ui/index.md

```diff
@@ -0,0 +1,55 @@
+---
+banner: ./images/building-accessible-apps-with-speed-using-chakra-ui.png
+slug: introducing-the-codesandbox-github-app
```

Comment:
The slug is the same of the previous post, conflicting the redirect.

Reply:
Updated - thanks!